src/vorta/views/repo_add_dialog.py

```diff
@@ -27,28 +27,29 @@ def __init__(self, parent=None):
         self.saveButton = self.buttonBox.button(QDialogButtonBox.StandardButton.Ok)
         self.saveButton.setText(self.tr("Add"))
 
+        if self.__class__ == AddRepoWindow:
```

Comment:
This is hacky. Is it possible to remove the PasswordInput from the layout in a subclass?